backend/typescript/services/implementations/campService.ts

```diff
@@ -85,6 +92,201 @@ class CampService implements ICampService {
     }
   }
 
+  async updateCampById(campId: string, camp: UpdateCampDTO): Promise<CampDTO> {
```

Comment:
we also have to handle updating the camp photo if that's changed - will create a separate task for this